packages/framework/autoload-coding-standard.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
```

Comment:
I'm not sure about name of this file. In `coding-standards` package it is called `easy-coding-standard.yml`. I would prefer to name it the same.

Reply:
to autoload-easy-coding-standards.yml